lib/diff_file.php

```diff
@@ -20,9 +20,9 @@ public function dropTable($tableName)
         $this->drop[] = $tableName;
     }
 
-    public function ensureColumn($tableName, rex_sql_column $column)
+    public function ensureColumn($tableName, rex_sql_column $column, $after = null)
```

Comment:
Aber auch nur so halb public, denn die Klasse ist als `@internal` markiert ;)

Wichtiger wäre der Parametername im Haupt-PR in redaxo/redaxo, denn dort heißt der aktuell auch `$after`.
Was wäre dein Vorschlag? Würde es dann in beiden PRs anpassen.